packages/cozy-harvest-lib/src/components/KonnectorSuggestionModal/DataTypes.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/transpiled/react/I18n'
+import Text from 'cozy-ui/transpiled/react/Text'
+import CompositeRow from 'cozy-ui/transpiled/react/CompositeRow'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+
+const DataTypes = ({ t, dataTypes, konnectorName }) => {
+  return dataTypes.length > 0 ? (
+    <>
+      <Text className="u-ta-center">
+        {t('suggestions.data', { name: konnectorName })}
+      </Text>
+      <ul className="u-w-100 u-nolist u-mt-0 u-ph-0 u-mb-half">
+        {dataTypes.map(dataType => (
+          <li key={dataType}>
+            <CompositeRow
+              primaryText={t(`dataType.${dataType}`)}
+              image={
+                <Icon
+                  size={32}
+                  icon={
+                    require('!svg-sprite-loader!../../assets/datatypes/icon-' +
```

Comment:
I'm not super happy with this, because it literally ships  with this in the transpiled source code and so the parent app will need `svg-sprite-loader`. It's pretty standard and it's included in cozy-scripts, but I still wouldn't mind a better way to do this…

Reply:
Can you have something passing the grep test? 

something like : 
```
const iconsByDataTypes = {
'tweet' : 'icon-tweet.svg'
}
```

It'll be easier to maintain and replace :)